shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1983,6 +1985,17 @@ void WebContents::DraggableRegionsChanged(
   draggable_region_ = DraggableRegionsToSkRegion(regions);
 }
 
+void WebContents::PrintCrossProcessSubframe(
+    content::WebContents* web_contents,
+    const gfx::Rect& rect,
+    int document_cookie,
+    content::RenderFrameHost* subframe_host) const {
+  auto* client = printing::PrintCompositeClient::FromWebContents(web_contents);
+  if (client) {
```

Comment:
_(nit, style)_

This can be shortened:

```suggestion
  if (auto* client = printing::PrintCompositeClient::FromWebContents(web_contents)) {
```